addons/web/static/src/views/list/list_controller.xml

```diff
@@ -42,7 +42,7 @@
 
                 <t t-set-slot="control-panel-additional-actions">
                     <CogMenu t-if="!hasSelectedRecords"/>
-                    <CogMenu t-elif="env.isSmall and (props.info.actionMenus or archInfo.headerButtons.length)" t-props="this.actionMenuProps" hasSelectedRecords="hasSelectedRecords">
+                    <CogMenu t-elif="env.isSmall and (props.info.actionMenus or archInfo.headerButtons.length)" t-props="this.actionMenuProps">
```

Comment:
You're not passing `hasSelectedRecords` anymore... Whats the reason ? Is it replaced by something else (if yes the prop and the CogMenu's code using it doesn't make sense)

Reply:
Probably a mistake, I'll set it again